docs/content/how-to/azure/create-azure-cluster_on_aks.md

```diff
@@ -0,0 +1,165 @@
+--name $CLUSTER_NAME \
+--azure-creds $AZURE_CREDS \
+--location ${LOCATION} \
+--node-pool-replicas 2 \
+--base-domain $AZURE_BASE_DOMAIN \
+--pull-secret $PULL_SECRET \
+--generate-ssh \
+--release-image quay.io/openshift-release-dev/ocp-release-nightly@sha256:b619707647800f7c382e7cb36e7b1026d82a576661274baffaf0585dd257fd1d \
+--external-dns-domain ${DNS_ZONE_NAME} \
+--resource-group-name ${RG} \
+--annotations hypershift.openshift.io/pod-security-admission-label-override=baseline \
+--annotations hypershift.openshift.io/certified-operators-catalog-image=registry.redhat.io/redhat/certified-operator-index@sha256:fc68a3445d274af8d3e7d27667ad3c1e085c228b46b7537beaad3d470257be3e \
+--annotations hypershift.openshift.io/community-operators-catalog-image=registry.redhat.io/redhat/community-operator-index@sha256:4a2e1962688618b5d442342f3c7a65a18a2cb014c9e66bb3484c687cfb941b90 \
+--annotations hypershift.openshift.io/redhat-marketplace-catalog-image=registry.redhat.io/redhat/redhat-marketplace-index@sha256:ed22b093d930cfbc52419d679114f86bd588263f8c4b3e6dfad86f7b8baf9844 \
+--annotations hypershift.openshift.io/redhat-operators-catalog-image=registry.redhat.io/redhat/redhat-operator-index@sha256:59b14156a8af87c0c969037713fc49be7294401b10668583839ff2e9b49c18d6 
+```
+
+You will also need to create a ConfigMap named `openshift-service-ca.crt` with the data from the `root-ca` ConfigMap, with the key `service-ca.crt`, until this PR is merged - https://github.com/openshift/cluster-network-operator/pull/2309.
+
+You may need to restart the cluster-network-operator pod after creating this.
+
+## Important Notes
+You should see these pods fail
+```
+% k get pods | grep -v Running
+NAME                                                  READY   STATUS                            RESTARTS   AGE
+cloud-network-config-controller-785944b4d7-29bfn      0/3     Init:CreateContainerConfigError   0          24h
+csi-snapshot-controller-cfb96bff7-m8kf2               0/1     CreateContainerConfigError        0          27h
+csi-snapshot-webhook-57f9799848-p57c8                 0/1     CreateContainerConfigError        0          27h
+```
+
+The `cloud-network-config-controller` will fail until this PR is merged - https://github.com/openshift/cluster-network-operator/pull/2307.
+The `csi` pods will fail until this PR is merged - https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/197.
+
+However, your nodes should join the NodePool and you should be able to login to your HostedCluster by getting the console URL and the kubeadmin password.
+```
+% k get nodepool -A
+NAMESPACE   NAME                   CLUSTER                DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION                                    UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+clusters    brcox-hypershift-arm   brcox-hypershift-arm   2               2               False         False        4.16.0-0.nightly-multi-2024-02-26-105325
+```
+
```

Comment:
Good to include destroy command as well.
```suggestion
## Delete the Azure Hosted Cluster
Run the following command to delete an hypershift cluster

`hypershift destroy cluster azure --name $CLUSTER_NAME --azure-creds $AZURE_CREDS --resource-group-name ${RG}`

```

Reply:
Added